CHANGES

```diff
@@ -9,6 +9,25 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Introduced a new method type and API, OSSL_SERIALIZER, to
+     represent generic serializers.  An implementation is expected to
+     be able to serialize an object associate with a given name (such
+     as an algorithm name for an asymmetric key) into forms given by
+     implementation properties.
+
+     Serializers are primarily used from inside libcrypto, through
+     calls to functions like EVP_PKEY_print_private(),
+     PEM_write_bio_PrivateKey() and similar.
+
+     Serializers are specified in such a way that they can be made to
+     directly handle provider side portion of an object, if this
```

Comment:
portions

Reply:
I actually think singular is fitting.  It's the provider side portion of *an object*, and from libcrypto's point of view, each object only has one portion.

Reply:
I've added "the", though, making it "directly handle the provider side portion of an object, ...", maybe that makes it clearer?